src/com/epam/javacourse/homework4/Main.java

```diff
@@ -0,0 +1,136 @@
+                            int height = scanner.nextInt();
+                            scanner.nextLine();
+                            Shrub newPlant = new Shrub(name, type, nativeRegion, height);
+                            greenhouse.addPlant(newPlant);
+                            System.out.println("New shrub added successfully.");
+                        }
+                        default -> System.out.println("Invalid plant type entered.");
+                    }
+                }
+                case 2 -> {
+                    System.out.println("Enter the name of the plant to buy:");
+                    String name = scanner.nextLine();
+
+                    greenhouse.buyPlant(name);
+                }
+                case 3 -> {
+                    System.out.println("Enter the name of the plant to update:");
+                    String name = scanner.nextLine();
+
+                    Plant plantToUpdate = null;
+                    for (Plant plant : greenhouse.getPlants()) {
+                        if (plant.getPlantName().equals(name)) {
+                            plantToUpdate = plant;
+                            break;
+                        }
+                    }
+
+                    if (plantToUpdate == null) {
+                        System.out.println("No plant with the given name found.");
+                    } else {
+                        System.out.println("Enter the new name for the plant:");
+                        String newName = scanner.nextLine();
+                        greenhouse.updatePlant(plantToUpdate, newName);
+                    }
+                }
+                case 4 -> {
+                    System.out.println("Enter the name of the plant to remove:");
+                    String name = scanner.nextLine();
+
+                    Plant plantToRemove = null;
+                    for (Plant plant : greenhouse.getPlants()) {
+                        if (plant.getPlantName().equals(name)) {
+                            plantToRemove = plant;
+                            break;
+                        }
+                    }
+
+                    if (plantToRemove == null) {
+                        System.out.println("No plant with the given name found.");
+                    } else {
+                        greenhouse.removePlant(plantToRemove);
+                        System.out.println("Plant removed successfully.");
+                    }
+                }
+                case 5 -> {
+                    greenhouse.waterPlants();
+                    System.out.println("All plants have been watered.");
+                }
+                case 6 -> {
+                    System.out.println("Enter the temperature to set:");
+                    double temperature = scanner.nextDouble();
+                    scanner.nextLine();
+
+                    greenhouse.setTemperature(temperature);
+                }
+                case 7 -> {
```

Comment:
This block does not work as expected for me.
`addPlants()` method used in case 1 adds plants to `availablePlants` but not to the `plants` list.
And I'm not able to see the recent plant I've added when I press 7.